### DISCUSSION TITLE
Accordion with grid content

### DISCUSSION THREAD

**Discussion by ChadGatling** [Unity-UI-Extensions/com.unity.uiextensions#454] [Q&A]
I am working with an accordion. I have most of it working but I ran into an issue with using a Grid Layout Group instead of a Vert or Horz. The Vert and Horz layout groups will control the size of their children just fine but the grid does not seem to have that ability. The grid does not have an option to control child height so the grid children will just sit there under the header overlapping everything and not be shrunk down. Is there anything I can do before I just make rows and columns myself out of vert and horz layout groups?

**Comment by SimonDarksideJ** [comment]
Can you log a Feature Request to track the request as that sounds like a new feature?   The original Accordion control, by design, only supported Horizontal or Vertical flow, as it was the natural expansion of the accordion group.

To add Grid support, we would need more detail and possible diagrams for how a grid layout would look both collapsed and expanded.
